firexapp/firex_subprocess.py

```diff
@@ -492,6 +511,13 @@ def _collect_stats():
 
                 # Inactivity timeout check
                 current_output_size = os.fstat(f.fileno()).st_size
+                for file in found_monitor_activity_files:
```

Comment:
this `file` is clobbering the input `file`

Reply:
Good catch! thank you!!